src/main/webapp/app/entities/iris/iris.model.ts

```diff
@@ -0,0 +1,42 @@
+import dayjs from 'dayjs';
+import { User } from 'app/core/user/user.model';
+import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
+
+export class IrisSession {
+    id: number;
+    exercise?: ProgrammingExercise;
+    user?: User;
+    messages: IrisMessage[];
+}
+
+export enum IrisSender {
+    SERVER = 'LLM',
```

Comment:
`SERVER` is not really correct here. `ARTEMIS` messages are generated by the server. `LLM` messages are generated by the LLM.

Though I would propose that you simply don't change the naming here, as it makes the server and client code harder to compare and therefore maintain. (This also goes for `ARTEMIS`)